core/templates/pages/topics-and-skills-dashboard-page/topics-list/topics-list.directive.html

```diff
@@ -63,25 +79,76 @@
 <style>
   topics-list .list-view-item {
     background: #fff;
-    margin: 20px 7.5px 0 12.5%;
-    padding: 10px 20px;
-    width: 75%;
+    box-shadow: none;
+    margin-top: 50px;
   }
 
   topics-list .dashboard-table {
     font-size: 15px;
     width: 100%;
   }
+  topics-list .topic-list-status {
+    position: relative;
+  }
+  topics-list .topic-list-status-text {
+    display: inline-block;
+    text-align: right;
+    width: 90%;
+  }
+  topics-list .topic-edit-options {
+    background-color: white;
```

Comment:
Use HEX values for colours (see [guide](https://github.com/oppia/oppia/wiki/Coding-style-guide#css)). Ditto below.

Reply:
Done.